src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -130,9 +131,12 @@ export class ResourceTreeAdapter implements ReactAdapter {
       path: "Gallery",
       type: NotebookContentItemType.File,
     };
-
+    let notebookText = ResourceTreeAdapter.MyNotebooksTitle;
```

Comment:
move this to a util? i can still see this logic repeated in a few places. have a phoenix util, we can put more util methods there later

Reply:
use the useNotebooks.get().NotebooksFolderName